demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+export const LocalMediaStreamMetrics: React.FC = () => {
+  const audioVideo = useAudioVideo();
+  const localAttendeeId = (audioVideo as any).audioVideoController
+    ?.realtimeController?.state?.localAttendeeId;
+  const {
+    audioPacketsSentFractionLossPercent,
+    audioPacketsReceivedFractionLossPercent,
+    availableIncomingBandwidth,
+    availableOutgoingBandwidth,
+    videoStreamMetrics
+  } = useMediaStreamMetrics();
+  const localVideoStreamMetrics = videoStreamMetrics[localAttendeeId];
+  const ssrcArray = localVideoStreamMetrics
+    ? Object.keys(localVideoStreamMetrics)
+    : [];
+  const isLocalAudioActive =
+    audioVideo &&
+    audioPacketsSentFractionLossPercent !== null &&
+    audioPacketsReceivedFractionLossPercent !== null;
+  const isLocalVideoActive = audioVideo && ssrcArray.length !== 0;
+  const hasBandwidthInfo =
+    availableIncomingBandwidth !== null && availableOutgoingBandwidth !== null;
+  return (
+    <StyledMediaMetricsWrapper>
+      {isLocalAudioActive && (
+        <>
+          <PopOverHeader title={'Audio statistics'} />
+          <MediaStatsList>
+            <MetricItem metricName="" metricValues={['Up', 'Down']} />
+            <MetricItem
+              metricName="1s Loss"
+              metricValues={[
+                audioPacketsSentFractionLossPercent !== null ? audioPacketsSentFractionLossPercent.toString() : "",
```

Comment:
Same as another comment down below:
`isLocalAudioActive` checks for this already right?

Reply:
Same here, typescript will show error of : Object is possibly "null"

Reply:
Got it! Thanks for the explanation.